node/silkworm/downloader/rpc/hand_shake.hpp

```diff
@@ -0,0 +1,33 @@
+/*
+   Copyright 2021 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#ifndef SILKWORM_HAND_SHAKE_HPP
```

Comment:
Include guards should be `SILKWORM_DOWNLOADER_RPC_HAND_SHAKE_HPP_`
If we don't include the full path we might end up with clashing includes